pkg/controlplane/apiserver/server.go

```diff
@@ -203,6 +203,10 @@ func (c completedConfig) New(name string, delegationTarget genericapiserver.Dele
 			c.Extra.PeerEndpointLeaseReconciler,
 			c.Extra.PeerEndpointReconcileInterval,
 			client.Cluster(LocalAdminCluster.Path()))
+		if err != nil {
+			return nil, fmt.Errorf("failed to create peer endpoint lease controller: %w", err)
+		}
+
```

Comment:
needs to be dropped. There is no error here anymore.

Reply:
Hmm seems to have been added by https://github.com/kcp-dev/kubernetes/commit/e142cc7d03344ee74a86902182b1821064ea3cbd by mistake? There doesn't seem to be anything creating an `err` there either. Thanks!